.travis.yml

```diff
@@ -34,11 +33,11 @@ matrix:
       dist: xenial
       env:
         - DEPENDS="$DEPENDS scipy"
-    # To test minimum dependencies for Python 3.6:
-    - python: 3.4
+    # To test minimum dependencies for Python 3.5:
+    - python: 3.5
       env:
         # Check these values against requirements.txt and dipy/info.py
-        - DEPENDS="cython==0.29 numpy==1.8.2 scipy==1.0 nibabel==2.4.0 h5py==2.4.0"
+        - DEPENDS="cython==0.29 numpy==1.9.0 scipy==1.0 nibabel==2.4.0 h5py==2.4.0"
```

Comment:
Finally, I have one comment: why do you update this Numpy version ? If we really need to update it, can you do it for `requirements.txt` and `dipy/info.py`

Reply:
Because there is no wheel for python 3.5/numpy 1.8.2.

Reply:
make sense, thanks!